components/Operations/tezConnections.ts

```diff
@@ -106,10 +104,20 @@ export function updateStakingOpsStatus(
           totalFinalizableAmount += operation.remainingFinalizableAmount
         }
       } else {
+        if (!prevBakerAddress) {
```

Comment:
I think `prevBakerAddress` is always gonna be null, isn't it? no need for this check